init_stop_sequences/init_stop_sequences.py

```diff
@@ -0,0 +1,59 @@
+from Communication.serial_functions.functions import *
+import Communication.tests.fake_serial as fake_ser
```

Comment:
FakeSerial should not be imported here. There should be a init_stop_sequences_test module where the FakeSerial will be imported.